common/openpype_common/distribution/addon_distribution.py

```diff
@@ -99,23 +142,29 @@ class HTTPAddonDownloader(AddonDownloader):
     CHUNK_SIZE = 100000
 
     @classmethod
-    def download(cls, source, destination_dir, data=None):
+    def download(cls, source, destination_dir, data):
         source_url = source["url"]
```

Comment:
Do we know if there could be an instance where `source ` is an empty dict?

Reply:
There can be, but in that case it is invalid source and would not get to `download` phase.